doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -183,6 +190,32 @@ Finally, if we want to cancel the job, we can use ``ray job stop``:
     # Job submission server address: http://127.0.0.1:8265
     # Job 'raysubmit_tUAuCKubPAEXh6CW' was stopped
 
+.. _jobs-remote-cluster
+
+Using a Remote Cluster
```

Comment:
I think this section was inadvertently deleted in the past, but it's still relevant.